src/Skyra.ts

```diff
@@ -1,6 +1,7 @@
+import { SkyraClient } from '@lib/SkyraClient';
+import { CLIENT_OPTIONS, TOKENS } from '@root/config';
+import 'module-alias/register';
```

Comment:
Shouldn't you register before importing with `@lib`?

Reply:
Fixed in b461c4e, I noticed it at the same time